src/edu/csus/ecs/pc2/core/model/Problem.java

```diff
@@ -2175,5 +2180,13 @@ public void setSandboxType(SandboxType sandboxType) {
             sandboxProgramName = Constants.PC2_INTERNAL_SANDBOX_PROGRAM_NAME;
         }
     }
+    
+    public boolean isOnLoadDataFilesLoadSamplesFirst() {
```

Comment:
Similarly to the comments (above) regarding naming of the variable, I think this method should be named (something like) `isLoadDataFilesSamplesFirst()`

Reply:
pushed fix.